crates/torii/cli/src/options.rs

```diff
@@ -372,6 +372,45 @@ impl Default for ErcOptions {
     }
 }
 
+#[derive(Debug, clap::Args, Clone, Serialize, Deserialize, PartialEq, Default)]
+#[command(next_help_heading = "SQL options")]
+pub struct SqlOptions {
+    /// Whether model tables should default to having indices on all columns
+    #[arg(
+        long = "sql.all_model_indices",
+        default_value_t = false,
+        help = "If true, creates indices on all columns of model tables by default. If false, \
+                only key fields columns of model tables will have indices."
+    )]
+    #[serde(default)]
+    pub all_model_indices: bool,
```

Comment:
May be more explicit to have an error if `all_model_indices` is provided, but the `sql.model_indices` is not empty. Maybe using `exclusive` to ensure both are not used at the same time? But we may be careful of also having this solved when merged with a configuration file.

Just to ensure that someone doesn't have a `all_model_indices` to true which is taking effect over the listed indices.